tests/formatURLInput.test.js

```diff
@@ -0,0 +1,19 @@
+const formatter = require('../utils/formatURLInput.js');
+
+test('Does nothing if input URL begins with http or https', () => {
```

Comment:
You could potentially be more specific here with "does nothing." For example, "Does not modify the input URL if it beings with http or https".

Reply:
Great suggestion!